unimol_tools/unimol_tools/config/model_config.py

```diff
@@ -1,16 +1,18 @@
 MODEL_CONFIG = {
     "weight":{
-        "protein": "pocket_pre_220816.pt",
```

Comment:
this should align with release version name, but i think it just an issue left over from different versions anyway.